recipes/install_binary.rb

```diff
@@ -8,9 +8,11 @@
 
 include_recipe 'ark'
 
+node.default['packer']['zipfile'] = "packer_#{node['packer']['version']}_" \
+  "#{node['os']}_#{node['packer']['arch']}.zip"
```

Comment:
A long time ago, there was an issue with using node attributes to generate values for other attributes inside the attributes directory. If I recall, it affected attributes that were overridden elsewhere, like in roles or environments. If that's no longer the case, then it should go into `attributes/default.rb`, absolutely.